junifer/data/masks.py

```diff
@@ -88,11 +146,66 @@ def list_masks() -> List[str]:
     return sorted(_available_masks.keys())
 
 
+def get_mask(
+    mask: Union[str, Dict],
+    target_data: Dict[str, Any],
+) -> "Nifti1Image":
+    """Get mask, tailored for the target image.
+
+    Parameters
+    ----------
+    masks : str or dict
```

Comment:
this should be str or list of (str or dict) now, no?

Reply:
for the moment this is for one mask at the time.